drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -544,7 +544,7 @@ void RasterizerStorageGLES2::texture_allocate(RID p_texture, int p_width, int p_
 	bool compressed = false;
 
 	if (p_flags & VS::TEXTURE_FLAG_USED_FOR_STREAMING) {
-		p_flags &= ~VS::TEXTURE_FLAG_MIPMAPS; // no mipies for video
+		p_flags &= ~((unsigned int)VS::TEXTURE_FLAG_MIPMAPS); // no mipies for video
```

Comment:
That's a bummer...

But I guess we don't have much choice. I looked up strongly-typed enums of C++11 and it would be the same issue as they explicit disable casting to `int` (and likely `unsigned int` if they're declared with this type).